src/Cake.Yaml/Cake.Yaml.csproj

```
@@ -14,13 +14,14 @@
     <Authors>Redth</Authors>
     <Owners>Redth,cake-contrib</Owners>
     <PackageProjectUrl>https://github.com/cake-contrib/Cake.Yaml</PackageProjectUrl>
-    <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/cake-contrib-medium.png</PackageIconUrl>
+    <PackageIcon>images/icon.png</PackageIcon>

```

```suggestion
    <PackageIcon>images/icon.png</PackageIcon>
    <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/addin/cake-contrib-addin-medium.png</PackageIconUrl>
    <NoWarn>$(NoWarn);NU5048</NoWarn>
```

Embedded icon is the way forward for sure, but we also keep the `PackageIconUrl` for devs that use NuGet clients that don't yet support embedded icons. Need to disable the `NU5048` warning as well (_The 'PackageIconUrl'/'iconUrl' element is deprecated_).

---

I've added back the PackageIconUrl.

---

And it seems it's not needed to suppress the NU5048 warning when the package contains both the PackageIcon and PackageIconUrl.
I've tested locally with my Cake.Recipe 2.0.0 build, no more warnings there.
And the [documentation](https://aka.ms/deprecateIconUrl) states that as long as the package has PackageIcon, you will not see the warning anymore, so I did not add the warning suppression as it's not needed.

---

Excellent. Thanks @escarls !